torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -763,16 +763,8 @@ std::shared_ptr<SugaredValue> toSugaredValue(
         // Register class
         auto rcb = py::module::import("torch._jit_internal")
                        .attr("createResolutionCallbackForClassMethods")(obj);
-
-        {
-          // We're starting a new compilation, so update the error call stack in
-          // case it fails
-          ErrorReport::CallStack stack(qualname.name());
```

Comment:
Does the error call stack still get updated somewhere else while doing this? It'd be hard to debug otherwise.

Reply:
Yes, the call stack gets set in `_recursive_compile_class`